lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -665,7 +665,7 @@ public static function get_site_data() {
 		}
 
 		/** This filter is documented in wp-includes/class-wp-theme-json-resolver.php */
-		$theme_json   = apply_filters( 'wp_theme_json_data_user', new WP_Theme_JSON_Data_Gutenberg( $config, 'site' ) );
+		$theme_json   = apply_filters( 'wp_theme_json_data_site', new WP_Theme_JSON_Data_Gutenberg( $config, 'site' ) );
```

Comment:
Could you explain a little about this change and how that filter works?

Reply:
The existing filters allow 3rd parties to modify each layer independently. They were introduced in WordPress 6.1, see devnote at https://make.wordpress.org/core/2022/10/10/filters-for-theme-json-data/

Because the `site` is a new "layer" in addition to the existing, it requires a separate filter.

Does this help?